src/Features/Core/Portable/QuickInfo/CommonSemanticQuickInfoProvider.cs

```diff
@@ -155,11 +156,8 @@ private static bool HasNoErrors(ImmutableArray<ISymbol> symbols)
             CancellationToken cancellationToken)
         {
             var descriptionService = workspace.Services.GetLanguageServices(token.Language).GetRequiredService<ISymbolDisplayService>();
-            var formatter = workspace.Services.GetLanguageServices(semanticModel.Language).GetRequiredService<IDocumentationCommentFormattingService>();
             var syntaxFactsService = workspace.Services.GetLanguageServices(semanticModel.Language).GetRequiredService<ISyntaxFactsService>();
 
-            var showWarningGlyph = supportedPlatforms != null && supportedPlatforms.HasValidAndInvalidProjects();
-
             var symbols = tokenInformation.Symbols;
```

Comment:
logic that cares abotu 'token' or 'tokenInformation' stayed here.  The values we need from those are then just passed into the common helper which was moved to a Utilities class.